migrations/20200915113220_van_result_codes.js

```diff
@@ -68,7 +68,8 @@ exports.up = function(knex) {
           'api_key', json_build_object('__secret', v_api_key_ref),
           'van_system_id', queue_refresh_van_result_codes.van_system_id,
           '__after', 'insert_van_result_codes'
-        )
+        ),
+        'van-api'
```

Comment:
This could cause a refresh to be delayed while a 100k sync runs - let's have:
`van-api-refresh` and `van-api-sync`